doc/source/how-to/setting-up.rst

```diff
@@ -0,0 +1,528 @@
+Setting Up Your Development Environment
+=======================================
+Before you can contribute to any ``PyAnsys`` project, you must set up
+your developer environment.
+
+Python
+------
+.. raw:: html
+    
+    <div align="center">
+      <img src="https://github.com/python/pythondotorg/raw/main/static/community_logos/python-logo-inkscape.svg">
+    </div>
+
+All ``PyAnsys`` projects require a Python interpreter for interacting
+with the software. Therefore, you must ensure that at least one Python
+interpreter is installed on your local machine.
+
+Installation
+~~~~~~~~~~~~
+There are multiple ways to install a Python package on your local machine:
+
+- Use an official installer from the `official Python download section <https://www.python.org/downloads/>`_.
+- Install via a package manager or "store" on your machine.
+
+.. warning:: 
+
+  Ensure that you install Python from an official channel. Do not trust
+   third-party websites or download executable content from them.
```

Comment:
```suggestion

  Ensure that you install Python from an official channel. Do not trust third-party websites or download executable content from them.
```

Reply:
Try writing the warning in a single line, otherwise the output looks kind of weird

![image](https://user-images.githubusercontent.com/37798125/171391281-b91210bf-b8f5-4214-8d9d-98e4124e491d.png)

Reply:
Thanks for pointing this, @RobPasMue. This is due to a bad indentation of the admonition content.

Reply:
Fixed in a5017d1